server/auth.go

```diff
@@ -693,3 +702,23 @@ func comparePasswords(serverPassword, clientPassword string) bool {
 	}
 	return true
 }
+
+func validateAuth(o *Options) error {
+	if o.NoAuthUser == "" {
+		return nil
+	}
+	if len(o.TrustedOperators) > 0 {
+		return fmt.Errorf("no_auth_user not compatible with Trusted Operator")
+	}
+	if o.Users == nil {
+		return fmt.Errorf(`no_auth_user: "%s" present, but users are not defined`, o.NoAuthUser)
```

Comment:
Detail, but you can use `%q` for a string that you want surrounded with `"`. With that you don't need the "`" sign for this string.

Reply:
will fix and squash.